lib/pyld/documentloader/requests.py

```diff
@@ -60,7 +60,22 @@ def loader(url, options={}):
                 headers = {
                     'Accept': 'application/ld+json, application/json'
                 }
-            response = requests.get(url, headers=headers, **kwargs)
+
+            if pieces.scheme == "ipfs":
```

Comment:
As I have already commented earlier, this should probably be refactored to a separate project.